.changeset/twelve-tigers-stare.md

```diff
@@ -0,0 +1,274 @@
+---
+"generated-test-app": patch
+"rainbowkit-next-app": patch
+"with-next-siwe-iron-session": patch
+"with-next-rainbow-button": patch
+"with-next-siwe-next-auth": patch
+"with-next-custom-button": patch
+"with-next-wallet-button": patch
+"with-create-react-app": patch
+"with-next-app-i18n": patch
+"with-next-mint-nft": patch
+"@rainbow-me/create-rainbowkit": patch
+"@rainbow-me/rainbow-button": patch
+"with-next-app": patch
+"with-remix": patch
+"@rainbow-me/rainbowkit": patch
+"with-next": patch
+"with-vite": patch
+"example": patch
+"site": patch
+---
+
+**Wagmi V2 Integration and EIP6963 Support**
```

Comment:
We should separate this into 2 pieces, first focussing on the Migration guidance, then discussing new features.
Check out these: 
- https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%401.0.0
- https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%400.8.0